plugins/modules/cloud/scaleway/scaleway_volume.py

```diff
@@ -134,13 +137,7 @@ def core(module):
         if volume['project'] == project and volume['name'] == name:
```

Comment:
@remyleone are there cases where `organization` is still returned (and `project` is `null`)?

Reply:
I don't think so.